lib/gnome.rb

```diff
@@ -0,0 +1,21 @@
+class GnomePostinstall
+  @gnome_postinstall = false
+
+  def self.activate
+    @gnome_postinstall = true
+  end
+
+  def self.run
+    if @gnome_postinstall
```

Comment:
I guess I'm not understanding why this is needed.  Can't we just run when needed simply with an `if @gnome` condition?

Reply:
We only want to run these once. If you're installing a bunch of gnome dependencies, you don't want the exact same postinstall code to run a dozen times. It only needs to run once, at the end.

Reply:
Then run it just once at the end with the `if @gnome` conditional.

Reply:
At the end of what exactly?